docs/source/challenge_creation.md

```diff
@@ -13,6 +13,8 @@ One can create a challenge in EvalAI using following methods:
 
 Creating a challenge on EvalAI is a three step process. You just need to upload the challenge details in a challenge configuration file (**YAML file**) and we will take care of the rest.
 
+The [link to repository](https://github.com/Cloud-CV/EvalAI-Examples) for an example to create a challenge, using 'zip_file'.
```

Comment:
@HargovindArora: Please remove it from here and add the folloiwng content to  https://github.com/Cloud-CV/EvalAI/pull/1185/files#diff-062fa4d29b790cecfd77766c9592e18eR186

```
### Challenge Creation Examples

Please see this [repository](https://github.com/Cloud-CV/EvalAI-Examples) to know how to create different types of challenges on EvalAI.
```